### PR TITLE
Fix typos, grammar, and broken links

### PR DESCRIPTION
**Pull request by pranjalkole**

### CODE REVIEW

File: How-to-setup-Anti-Captcha.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ editor: markdown
...
-2. Head over the "Add fund" page: https://anti-captcha.com/clients/finance/refill and use whatever currency you prefer to use for adding $2 into your balance.   
-Do note that cryptocurrencies give a bonus of 20% at the first time you add funds then 10% for each new recharge.
-3. After adding funds, go the API key link: https://anti-captcha.com/clients/settings/apisetup and copy your "Account key" (**never share it, it's confidential**).
-4. Edit your Invidious `config.yml` file (should be located in the `config` directory) and add your account key at the end of the file after the `captcha_key` parameter.   
-Here is an example:   
+1. [Register on anti-captcha.com](https://anti-captcha.com/clients/entrance/register) and finish the registration with the link provided in your mailbox.
+2. Head over to the [Add fund](https://anti-captcha.com/clients/finance/refill) page and use whatever currency you prefer to use for adding $2 into your balance.
+Do note that cryptocurrencies give a bonus of 20% for the first time you add funds and then 10% for each new recharge.
```
Comment (SamantazFox) [inline How-to-setup-Anti-Captcha.md:13]: "20% for the first time": "for" can be removed

**Review by SamantazFox — CHANGES_REQUESTED** [review]
Looks good to me, except for the `Installation.md` and `` files: can you please undo you changes on those two files? The former is being rewritten in #160, and the second is currently parsed by the instances API, which feeds the redirector, so most format changes in here can break https://redirect.invidious.io.

**Comment by pranjalkole** [comment]
I'm not very familiar with GitHub's pull request UI, but I think commits can be squashed while merging a pull request. If so, please squash the two commits.

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
Thanks for your contribution :)

**Pull request merged**